app/views/UploadsView.php

```diff
@@ -162,7 +162,14 @@ public function buildUploads($mode)
 
             // if this is something 3Dmol.js can handle
             } elseif (in_array($ext, $molExtensions)) {
-                $molviewer = new MolViewer($upload['id'], $filepath);
+                // try to be clever and choose stick representation for
+                // all files that are not in pdb format
+                if ($ext === 'pdb') {
+                  $style = 'cartoon:color=spectrum';
+                } else {
```

Comment:
Can you refactor without 'else' please ?
$style = 'stick'
if (something) {
 style = somethingelse
}

Thank you :)